src/tsickle.ts

```diff
@@ -105,33 +105,61 @@ class ClosureRewriter extends Rewriter {
     super(file);
   }
 
-  emitFunctionType(fnDecl: ts.SignatureDeclaration, extraTags: jsdoc.Tag[] = []) {
+  /**
+   * Handles emittng the jsdoc for methods, including overloads.
+   * If overloaded, merges the signatures in the list of SignatureDeclarations into a single jsdoc.
+   * - Total number of parameters will be the maximum count found across all variants.
+   * - Any parameters beyond the minimum count found across all variants; prefix: "opt_"
+   * - Different names at the same parameter index will be joined with "_or_"
+   * - Variable args (...type[] in TypeScript) will be output as "...type",
+   *    except if found at the same index as another argument.
+   * @param {Array<ts.SignatureDeclaration>} fnDecls Pass > 1 declaration for overloads of same name
```

Comment:
Tsickle itself actually rejects input code that specifies a type here.  :)
You can write it without the type, like
@param fnDecls Pass > 1 declaration...
same with @return.

(Also you don't need to document everything, just whatever you think isn't obvious.  So you can drop extraTags unless you want to write something about it.)